src/extension/sldService/src/main/java/org/geoserver/sldservice/utils/classifier/RulesBuilder.java

```diff
@@ -315,7 +325,8 @@ public StyleFactory getStyleFactory() {
         Filter f;
         List<Rule> list = new ArrayList();
         Expression att = normalizeProperty(FF.property(property), propertyType, normalize);
-
+        PercentagesManager percMan = null;
```

Comment:
Why split declaration and assignment over two lines?

Reply:
fixed